code/modules/reagents/chemistry/machinery/chem_heater.dm

```diff
@@ -30,10 +37,12 @@
 				on = FALSE
 				SSnanoui.update_uis(src)
 				return
-			beaker.reagents.temperature_reagents(desired_temp)
-			beaker.reagents.temperature_reagents(desired_temp)
-			if(abs(beaker.reagents.chem_temp - desired_temp) <= 3)
+			beaker.reagents.temperature_reagents(desired_temp, 35 - speed_increase)
```

Comment:
Should not happen. But maybe add a `max` here to ensure that the divisor is never 0

Reply:
Rookie mistake from my part, done